python/tskit/trees.py

```diff
@@ -33,6 +33,7 @@
 import concurrent.futures
 
 import numpy as np
+import textwrap
```

Comment:
textwrap is part of the standard library, so it belongs in the first group of imports.